navigator/backend/BUILD.bazel

```diff
@@ -62,6 +62,7 @@ da_scala_library(
         "//ledger/ledger-api-domain",
         "//libs-scala/build-info",
         "//libs-scala/grpc-utils",
+        "//libs-scala/scala-utils",
```

Comment:
Uh? I'm a bit surprised we're adding a dependency here. I guess I'll see why later in the diff.

Reply:
Ah, `discard`, I see.